src/components/mobile/mobileAssetView.tsx

```diff
@@ -0,0 +1,302 @@
+                <tr>
+                  <th>Network Port</th>
+                  <th>Mac Address</th>
+                  <th>Destination Asset</th>
+                  <th>Destination Port</th>
+                </tr>
+                <tbody>
+                  {this.props.asset.model.network_ports.map((port: string) => {
+                    var connection = this.getNetworkConnectionForPort(port);
+                    return (
+                      <tr>
+                        {" "}
+                        <td style={getChangePlanRowStyle(this.props.asset)}>
+                          {port}
+                        </td>
+                        <td style={getChangePlanRowStyle(this.props.asset)}>
+                          {this.props.asset.mac_addresses
+                            ? this.props.asset.mac_addresses[port]
+                            : null}
+                        </td>{" "}
+                        {connection
+                          ? [
+                              <td
+                                style={getChangePlanRowStyle(this.props.asset)}
+                                className={
+                                  this.props.asset.decommissioning_user
+                                    ? undefined
+                                    : "asset-link"
+                                }
+                                // onClick={
+                                //   this.props.asset.decommissioning_user
+                                //     ? undefined
+                                //     : (e: any) => {
+                                //         const id = this.getAssetIdFromHostname(
+                                //           connection!.destination_hostname!
+                                //         );
+                                //         if (id) {
+                                //           this.redirectToAsset(id);
+                                //         }
+                                //       }
+                                // }
+                              >
+                                {connection.destination_hostname}
+                              </td>,
+                              <td
+                                style={getChangePlanRowStyle(this.props.asset)}
+                              >
+                                {connection.destination_port}
+                              </td>,
+                            ]
+                          : [<td></td>, <td></td>]}
+                      </tr>
+                    );
+                  })}
+                </tbody>
+              </table>
+
+              <NetworkGraph
+                networkGraph={this.props.asset.network_graph}
+                onClickNode={() => {}}
+                isDecommissioned={
+                  this.props.asset.decommissioning_user !== undefined
+                }
+              />
```

Comment:
This does not render on mobile. Maybe it's a platform issue where the Network Graph component is just not compatible, idk. Do you think we should show this or remove it? The requirements again do not specify other than to say that we should show "pertinent information about the asset"

Reply:
good piazza Q 😁